### PR TITLE
update URI examples

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/docs/pull/15165

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

File: external-storage-uri.md [hunk 1]
```diff
@@ -30,10 +30,16 @@ summary: 介绍了外部存储服务 Amazon S3、GCS、和 Azure Blob Storage 
...
 
-示例：
+以下是用于 TiDB Lightning 和 BR 的 Amazon S3 URI 示例，需要指定文件夹路径 `testfolder`：
 
 ```shell
-s3://external/backup-20220915?access-key=${access-key}&secret-access-key=${secret-access-key}"
+s3://external/testfolder?access-key=${access-key}&secret-access-key=${secret-access-key}"
+```
+
+以下是用于 [IMPORT INTO](/sql-statements/sql-statement-import-into.md) 的 Amazon S3 URI 示例，需要指定具体的文件名 `test.csv`：
```
Comment (lilin90) [inline external-storage-uri.md:39]:
```suggestion
以下是用于 [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) 的 Amazon S3 URI 示例，需要指定具体的文件名 `test.csv`：
```

File: external-storage-uri.md [hunk 2]
```diff
@@ -46,10 +52,16 @@ s3://external/backup-20220915?access-key=${access-key}&secret-access-key=${secre
...
 
-示例：
+以下是用于 TiDB Lightning 和 BR 的 GCS URI 示例，需要指定文件夹路径 `testfolder`：
 
 ```shell
-gcs://external/backup-20220915?credentials-file=${credentials-file-path}
+gcs://external/testfolder?credentials-file=${credentials-file-path}
+```
+
+以下是用于 [IMPORT INTO](/sql-statements/sql-statement-import-into.md) 的 GCS URI 示例，需要指定具体的文件名 `test.csv`：
```
Comment (lilin90) [inline external-storage-uri.md:61]:
```suggestion
以下是用于 [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) 的 GCS URI 示例，需要指定具体的文件名 `test.csv`：
```

File: external-storage-uri.md [hunk 3]
```diff
@@ -65,8 +77,14 @@ gcs://external/backup-20220915?credentials-file=${credentials-file-path}
...
 
-示例：
+以下是用于 TiDB Lightning 和 BR 的 Azure Blob Storage URI 示例，需要指定文件夹路径 `testfolder`：
 
 ```shell
-azure://external/backup-20220915account-name=${account-name}&account-key=${account-key}
+azure://external/testfolder?account-name=${account-name}&account-key=${account-key}
 ```
+
+以下是用于 [IMPORT INTO](/sql-statements/sql-statement-import-into.md) 的 Azure Blob Storage URI 示例，需要指定具体的文件名 `test.csv`：
```
Comment (lilin90) [inline external-storage-uri.md:86]:
```suggestion
以下是用于 [`IMPORT INTO`](/sql-statements/sql-statement-import-into.md) 的 Azure Blob Storage URI 示例，需要指定具体的文件名 `test.csv`：
```

**Review by lilin90 — COMMENTED** [review]
/lgtm

**Pull request merged**